docs/source/migration.rst

```diff
@@ -0,0 +1,14 @@
+.. _migration:
+
+Migrating from pyQuil v2 to v3
+==============================
+
+Most of the changes to pyQuil in v3 were under the hood. There are, however, a few user-facing changes that might affect you.
```

Comment:
These first two sentences probably aren't valuable to users - I would consider dropping.